webpack/scenes/Content/ContentPage.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Grid, GridItem, TextContent, Text, TextVariants } from '@patternfly/react-core';
+import { isEmpty } from 'lodash';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { STATUS } from 'foremanReact/constants';
+import ContentTable from './Table/ContentTable';
+import { selectContentTypes, selectContentTypesStatus } from './ContentSelectors';
+import ContentConfig from './ContentConfig';
+import { getContentTypes } from './ContentActions';
+import Loading from '../../components/Loading';
+import EmptyStateMessage from '../../components/Table/EmptyStateMessage';
+
+const ContentPage = () => {
+  const dispatch = useDispatch();
+  const contentTypesResponse = useSelector(selectContentTypes);
+  const contentTypesStatus = useSelector(selectContentTypesStatus);
+  const [selectedContentType, setSelectedContentType] = useState(null);
+  const [contentTypes, setContentTypes] = useState(null);
+
+  function buildContentTypes(response) {
```

Comment:
```suggestion
  const buildContentTypes = () => {
```
The param shouldn't need to be passed here. Can directly use the contentTypesResponse

Reply:
also! this is es5 syntax, we tend to want to avoid that in our javascript react code now since about 2012. 
Here's an in depth breakdown of what I mean: 
[es5 vs es6 functions (the big jump)](https://engineering.carsguide.com.au/es5-vs-es6-syntax-6c8350fa6998)
[all the more modern methods](https://www.freecodecamp.org/news/es5-to-esnext-heres-every-feature-added-to-javascript-since-2015-d0c255e13c6e/)